policy-controller/src/admission.rs

```diff
@@ -0,0 +1,80 @@
+use super::api::policy::{Server, ServerSpec};
+use kube::api::{Api, ListParams};
+use kube::core::{
+    admission::{AdmissionRequest, AdmissionResponse, AdmissionReview},
+    DynamicObject, ObjectList,
+};
+use std::convert::{Infallible, TryInto};
+use std::fmt;
+use tracing::{error, info, instrument};
+use warp::{reply, Reply};
+
+#[derive(Clone)]
+pub struct Admission(pub kube::Client);
+
+// A general /mutate handler, handling errors from the underlying business logic
```

Comment:
Is this comment from the example? Are we actually doing mutation? Do we serve a `/mutate` path? better to either remove the comment or add a proper `/// Doc comment`

Also should this be named something else if its not a mutating webhook?